--- a/src/bignum.cc
+++ b/src/bignum.cc
@@ -88,10 +88,10 @@
 
 
 static uint64_t ReadUInt64(Vector<const char> buffer,
-                           int from,
+                           unsigned int from,
                            int digits_to_read) {
   uint64_t result = 0;
-  for (int i = from; i < from + digits_to_read; ++i) {
+  for (unsigned int i = from; i < from + digits_to_read; ++i) {
     int digit = buffer[i] - '0';
     ASSERT(0 <= digit && digit <= 9);
     result = result * 10 + digit;
@@ -105,7 +105,7 @@
   const int kMaxUint64DecimalDigits = 19;
   Zero();
   int length = value.length();
-  int pos = 0;
+  unsigned int pos = 0;
   // Let's just say that each digit needs 4 bits.
   while (length >= kMaxUint64DecimalDigits) {
     uint64_t digits = ReadUInt64(value, pos, kMaxUint64DecimalDigits);
